library/tasks/ScatterBam.wdl

```diff
@@ -1,18 +1,22 @@
-task ScatterBam {
+version 1.0
 
-  File bam_to_scatter
-  Int scatter_width
+task ScatterBam {
+  input {
+    File bam_to_scatter
+    Int scatter_width
+  }
+  
   Int disk_size = ceil(size(bam_to_scatter, "GiB") * 3)
 
-  command <<<
+  command {
     mkdir scattered_bams
     java -Xms7g -jar /usr/picard/picard.jar \
       SplitSamByNumberOfReads \
       INPUT=${bam_to_scatter} \
       SPLIT_TO_N_FILES=${scatter_width} \
-      OUT_PREFIX=$(basename ${bam_to_scatter} .bam)_split \
+      OUT_PREFIX=${basename(bam_to_scatter, '.bam')}_split \
```

Comment:
This looks incorrect

Reply:
Yes I think they changed that. Not sure why. Seems to be wroking fine. I think they are usintfuhe wdl basename function. @pshapiro4broad  ??